terraform/batch.tf

```diff
@@ -64,6 +65,8 @@ resource "aws_batch_job_definition" "job_definition" {
       logDriver = "awslogs"
     }
 
+    propagate_tags = true
```

Comment:
This is important so that the EC2 instance gets tagged and be costs can be tracked.